content/01/MAIN_tutorial_intro_to_nilearn.ipynb

```diff
@@ -345,6 +344,8 @@
    ],
    "source": [
     "from nilearn import plotting\n",
+    "%matplotlib inline\n",
```

Comment:
Did we figure out what in the new environment is causing this ?

Reply:
Did not investigated it, but mostly the way jupyter notebook is executing the code is the issue

Reply:
nilearn does not see `%matplotlib inline` if he is not imported before ??
Also in general I think it is a better practice to put all the import at the first cell, instead of having few in the notebook

Reply:
It wasn't an issue in other environments (like mybinder.org) ... I'm fine to merge as is but I think this is something we'll need to nail down.